core/src/main/java/org/jdbi/v3/core/mapper/MapMapper.java

```diff
@@ -51,7 +51,7 @@ public MapMapper() {
      */
     // TODO deprecate when MapMappers.caseChange is out of beta
```

Comment:
Ok, I could have sworn this wasn't deprecated yet. We should remove the release note that says this constructor was deprecated.